composer/lib/dependabot/composer/file_fetcher.rb

```diff
@@ -20,7 +20,9 @@ def self.required_files_message
       def ecosystem_versions
         {
           package_managers: {
-            "composer" => Helpers.composer_version(parsed_composer_json, parsed_lockfile) || "unknown"
+            composer: {
+              "max" => Helpers.composer_version(parsed_composer_json, parsed_lockfile)
```

Comment:
Today this method returns only the major version, but long term it'll return the full string, so we'll need to coerce it then from semver to a decimal of major.minor